modules/gapi/samples/onevpl_source_to_bgr_conv.cpp

```diff
@@ -0,0 +1,106 @@
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <cctype>
+#include <tuple>
+#include <memory>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/gpu/ggpukernel.hpp>
+#include <opencv2/gapi/streaming/onevpl/source.hpp>
+#include <opencv2/gapi/streaming/onevpl/data_provider_interface.hpp>
+#include <opencv2/gapi/streaming/onevpl/default.hpp>
+#include <opencv2/highgui.hpp> // CommandLineParser
+#include <opencv2/gapi/ocl/core.hpp>
+
+const std::string about =
+    "This is an example presents decoding on GPU using VPL Source and passing it to OpenCL backend";
+const std::string keys =
+    "{ h help       |                                                                | Print this help message }"
+    "{ input        |                                                                | Path to the input video file. Use .avi extension }"
+    "{ accel_mode   | mfxImplDescription.AccelerationMode:MFX_ACCEL_MODE_VIA_D3D11   | Acceleration mode for VPL }";
```

Comment:
one of 'accel' is redundant here:
```
accel_mode |  MFX_ACCEL_MODE_VIA_D3D11   
```
or
```
mfxImplDescription.AccelerationMode | MFX_ACCEL_MODE_VIA_D3D11   
```

or 
```
const std::string keys = 
   "..." +
   "{ " + onevpl::CfgParams::accel_name() + | MFX_ACCEL_MODE_VIA_D3D11 | Acceleration mode";
```

Reply:
For some reason it doesn't work w/o `mfxImplDescription.AccelerationMode:MFX_ACCEL_MODE_VIA_D3D11`